ui/app/services/flags.ts

```diff
@@ -60,9 +62,9 @@ export default class flagsService extends Service {
   }
 
   getActivatedFlags = keepLatestTask(async () => {
-    if (this.version.isCommunity) return;
     // Response could change between user sessions.
     // Fire off endpoint without checking if activated features are already set.
+    if (this.version.isCommunity) return;
```

Comment:
Edge case question - if a customer downgrades from enterprise, are their activated flags preserved? (Not necessary for this PR, mostly just curious)

Reply:
I think I've asked Robert this, and more or less no. I think because there's a conditionally wrapping that endpoint which is checking for enterprise, so it would return no. That being said I haven't tested this workflow.